### PR TITLE
2687 layout grid list

### PR DESCRIPTION
**Pull request by AlinaSelyutina**

### CODE REVIEW

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/layout/GridList.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.ISetup;
+import com.epam.jdi.light.material.annotations.JDIGridList;
+import com.epam.jdi.light.material.asserts.layout.GridListAssert;
+
+import java.lang.reflect.Field;
+
+public class GridList extends UIBaseElement<GridListAssert> implements ISetup {
```
Comment (kborovinsky25) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/elements/layout/GridList.java:14]: Please remove all methods. Add method for getting Section.

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/layout/GridListAssert.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.epam.jdi.light.material.asserts.layout;
+
+import static com.epam.jdi.light.asserts.core.SoftAssert.jdiAssert;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.material.elements.layout.GridList;
+import org.hamcrest.Matchers;
+
+public class GridListAssert extends UIAssert<GridListAssert, GridList> {
```
Comment (kborovinsky25) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/layout/GridListAssert.java:10]: Please remove GridList assertions

File: jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/annotations/JDIGridList.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.MarkupLocator;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE, ElementType.FIELD})
+public @interface JDIGridList {
```
Comment (kborovinsky25) [inline jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/annotations/JDIGridList.java:12]: Please remove custom annotation

File: jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/layout/GridListTests.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import static io.github.com.StaticSite.gridListFrame;
+import static io.github.com.StaticSite.layoutGridListPage;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.Test;
+
+public class GridListTests extends TestsInit {
+
+    @Test
+    public void gridListTest() {
```
Comment (kborovinsky25) [inline jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/layout/GridListTests.java:12]: Can we check that we can get section by index and check that found section is displayed

**Comment by AlinaSelyutina** [comment]
After discussion we decided  not implement this element. This is set of existing elements.

**Pull request closed without merging**